From f39f033d280be78db6012a57b6f0484af74d66ef Mon Sep 17 00:00:00 2001
From: "nirav.desai.843" <nirav.p.desai@slscorp.com>
Date: Wed, 21 Jun 2023 18:27:54 +0530
Subject: [PATCH] adding file

---
 file.txt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 file.txt

diff --git a/file.txt b/file.txt
new file mode 100644
index 0000000..497c512
--- /dev/null
+++ b/file.txt
@@ -0,0 +1 @@
+Adding new file
-- 
2.40.1.windows.1

